### DISCUSSION TITLE
Logistic terminal code line?

### DISCUSSION THREAD

**Discussion by Jaz0314** [Vdauphin/HeartsAndMinds#1612] [Question and answer]
Where in the files is the code for the logistics box and spawn circle? I wanted to try and make a second point on my map where you can spawn supplies and repair wrecks.

**Comment by Vdauphin** [comment]
Hello,

Just add variables names here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/fnc/int/add_actions.sqf#L119

like this
```sqf
} forEach [[btc_create_object, btc_create_object_point], [btc_create_object_1, btc_create_object_point_1]];
```

For reference
https://github.com/Vdauphin/HeartsAndMinds/discussions/1092

Cheers
